documentation/modules/ROOT/pages/connectors/db2.adoc

```diff
@@ -0,0 +1,1503 @@
+
+
+[[monitoring]]
+=== Monitoring
+
+Kafka, Zookeeper, and Kafka Connect all have xref:operations/monitoring.adoc[built-in support] for JMX metrics. The Db2 connector also publishes a number of metrics about the connector's activities that can be monitored through JMX. The connector has two types of metrics. Snapshot metrics help you monitor the snapshot activity and are available when the connector is performing a snapshot. Streaming metrics help you monitor the progress and activity while the connector reads CDC table data.
+
+[[monitoring-snapshots]]
+[[snapshot-metrics]]
+==== Snapshot Metrics
+
+===== *MBean: debezium.sql_server:type=connector-metrics,context=snapshot,server=_<database.server.name>_*
+
+[cols="30%a,10%a,60%a",width=100,options="header,footer",role="table table-bordered table-striped"]
+|=======================
+|Attribute Name
+|Type
+|Description
+
+|`LastEvent`
+|`string`
+|The last snapshot event that the connector has read.
+
+|`MilliSecondsSinceLastEvent`
+|`long`
+|The number of milliseconds since the connector has read and processed the most recent event.
+
+|`TotalNumberOfEventsSeen`
+|`long`
+|The total number of events that this connector has seen since last started or reset.
+
+|`NumberOfEventsFiltered`
+|`long`
+|The number of events that have been filtered by whitelist or blacklist filtering rules configured on the connector.
+
+|`MonitoredTables`
+|`string[]`
+|The list of tables that are monitored by the connector.
+
+|`QueueTotalCapcity`
+|`int`
+|The length of the queue used to pass events between the snapshotter and the main Kafka Connect loop.
+
+|`QueueRemainingCapcity`
+|`int`
+|The free capacity of the queue used to pass events between the snapshotter and the main Kafka Connect loop.
+
+|`TotalTableCount`
+|`int`
+|The total number of tables that are being included in the snapshot.
+
+|`RemainingTableCount`
+|`int`
+|The number of tables that the snapshot has yet to copy.
+
+|`SnapshotRunning`
+|`boolean`
+|Whether the snapshot was started.
+
+|`SnapshotAborted`
+|`boolean`
+|Whether the snapshot was aborted.
+
+|`SnapshotCompleted`
+|`boolean`
+|Whether the snapshot completed.
+
+|`SnapshotDurationInSeconds`
+|`long`
+|The total number of seconds that the snapshot has taken so far, even if not complete.
+
+|`RowsScanned`
+|`Map<String, Long>`
+|Map containing the number of rows scanned for each table in the snapshot. Tables are incrementally added to the Map during processing. Updates every 10,000 rows scanned and upon completing a table.
+
+|=======================
+
+
+[[monitoring-streaming]]
+[[streaming-metrics]]
+==== Streaming Metrics
+
+===== *MBean: debezium.sql_server:type=connector-metrics,context=streaming,server=_<database.server.name>_*
+
+[cols="30%a,10%a,60%a",width=100,options="header,footer",role="table table-bordered table-striped"]
```

Comment:
@Naros, does this need updating as per your recent change? This may have overlapped with this PR which I only got to review now.